### DISCUSSION TITLE
Unable to create user

### DISCUSSION THREAD

**Discussion by mountiny** [supabase/supabase#465] [Questions] [answered]
Hello! I have not been working on my project for two days and as I wanted to continue today, I found out I cannot signUp/create a new user. I can signIn, but signUp endpoint returns error saying "Database error saving new user".

I tried to search what might have caused this but unsuccessfully. I am not aware of changing any settings. Also, when trying to Invite a user in the Supabase Authentication dashboard, I get an error "Inviting user failed". 

Do you know what might be the reason behind these errors? I checked the status page and it seems all the Supabase servers should be up and running. 

Thank you very much for your help!

**Comment by silentworks** [comment]
@mountiny try logging out and logging back in, maybe your token has expired. Same in your app, try and delete the old token from `localStorage` and then try the signUp endpoint again.

**Reply by mountiny** [reply]
Hi @silentworks Thank you for the tip! I tried it, I cleared localStorage, I have logged out of the dashboard and cleared localStorage and the problem prevails. I have tried to signUp in different browser in which I have not had the project open or even the supabase dashboard and it does not work.

**Comment by awalias** [comment, marked as answer]
hey @mountiny have you played around with the `auth.users` table schema at all? We usually see this type of error when the table/column has accidentally been dropped

**Reply by mountiny** [reply]
@awalias Hi! I have used the Next.js / Stripe starter and so I had already the public users table where I have added a column and then I have dropped that column. Would this cause the problem? I thought this table is more "safe" than the auth table. Thanks!

**Reply by awalias** [reply]
ah yes looks like gotrue is choking on `ERROR: column \"full_name\" of relation \"users\" does not exist`

**Reply by mountiny** [reply]
@awalias That did it! i have added column "name" as I am not interested in collecting "full_name" from the users. I added the full_name column back and it is working again. Thank you very much! I would not be able to solve this! If it is not mentioned in the Stripe starter project, it probably should :D or I am sorry for misreading it.

Thank you once more. The problem was "full_name" column dropped in favour of "name".

**Reply by awalias** [reply]
Ah yes I think we probably need some more guidance on the role of the auth schema 👍 

Glad you got it working!

**Reply by mozeryansky** [reply]
> ah yes looks like gotrue is choking on `ERROR: column \"full_name\" of relation \"users\" does not exist`

How would I be able to find this message? I'm having an issue with creating users. (I've filed a ticket)

**Reply by munkyfoot** [reply]
I solved the issue by modifying the `handle_new_user` function in the Database > Functions section of the dashboard after modifying the public.users table. In my case, I removed `full_name` and replaced it with a `first_name` and `last name` column. Here's how the `handle_new_users` function needed to be modified for my case.

Before:
```
begin
  insert into public.users (id, full_name, avatar_url)
  values (new.id, new.raw_user_meta_data->>'full_name', new.raw_user_meta_data->>'avatar_url');
  return new;
end;
```

After:
```
begin
  insert into public.users (id, first_name, last_name, avatar_url)
  values (new.id, new.raw_user_meta_data->>'first_name', new.raw_user_meta_data->>'last_name', new.raw_user_meta_data->>'avatar_url');
  return new;
end;
```

@mozeryansky Not totally sure if this will solve your issue, but hopefully it helps.
